pyanaconda/ui/tui/spokes/user.py

```diff
@@ -193,14 +238,15 @@ def mandatory(self):
 
     @property
     def status(self):
-        if len(self.data.user.userList) == 0:
+        user_list = _get_user_list(self._users_module)
+        if len(user_list) == 0:
             return _("No user will be created")
-        elif self._use_password and not bool(self._user_data.password or self._user_data.isCrypted):
+        elif self._use_password and not bool(self._user_data.password or self._user_data.is_crypted):
             return _("You must set a password")
-        elif "wheel" in self.data.user.userList[0].groups:
-            return _("Administrator %s will be created") % self.data.user.userList[0].name
+        elif "wheel" in user_list[0].groups:
```

Comment:
Yeah, we do this in a couple places, let's do it. Maybe it could be a R/W property so you can turn the admin status on and off (we also do that in the user spoke).

Reply:
Good, point, added.